content/cpp/concepts/math-functions/terms/pow/pow.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'pow()'
+Description: 'Returns the result of a base raised to the power of an exponent.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`pow()`** function returns the result of a base raised to the power of an exponent.
+
+## Syntax
+
+```cpp
+pow(base, exponent)
+```
+
```

Comment:
Can we put some text below the syntax snippet that describes the domain and range of `base` and `exponent` as well as what errors are caused by which invalid input(s)?